parcels/examples/example_delay_start.py

```diff
@@ -1,59 +0,0 @@
-from parcels import FieldSet, ParticleSet, JITParticle, ScipyParticle
-from parcels import AdvectionRK4
-import numpy as np
-from datetime import timedelta as delta
-import pytest
-from netCDF4 import Dataset
-from os import path
-
-
-ptype = {'scipy': ScipyParticle, 'jit': JITParticle}
-
-
-@pytest.mark.parametrize('mode', ['scipy', 'jit'])
-def test_delay_start_example(mode, npart=10, show_movie=False):
-    """Example script that shows how to 'delay' the start of particle advection.
```

Comment:
Do you mean you don't show any example on how to add particles with pset.add anymore?
If it is the case, I think we should still show it. Indeed, if the user wants to add the particle with some variable depending on the result of what is going on, it still needs to use this pset.add

Reply:
Good point; we indeed do not document to `pset.add()` functionality anymore, except for in the test scripts, and in the online documentation at http://oceanparcels.org/parcels.html#parcels.particleset.ParticleSet.add. 
But then again, there is a lot of functionality that is not shown (yet) in examples. I wanted to remove this specific file because with this PR it makes no sense anymore to delay the start of particles through `pset.add()` statements and could only lead to confusion. Plus, there is already the delaystart jupyter notebook at http://nbviewer.jupyter.org/github/OceanParcels/parcels/blob/master/parcels/examples/tutorial_delaystart.ipynb

Reply:
Ok, no problem!